src/kontrol/prove.py

```diff
@@ -638,6 +664,27 @@ def _final_term(empty_config: KInner, contract_name: str, use_init_code: bool =
         'ADDRESSSET_CELL': KVariable('ADDRESSSET_FINAL'),
         'STORAGESLOTSET_CELL': KVariable('STORAGESLOTSET_FINAL'),
     }
+    if not is_test:
+        contract = KEVM.account_cell(
+            Foundry.address_TEST_SYMBOLIC(),
+            KVariable('CONTRACT_BAL_FINAL'),
+            program,
+            KVariable('CONTRACT_STORAGE_FINAL'),
+            KVariable('CONTRACT_ORIG_STORAGE_FINAL'),
+            KVariable('CONTRACT_NONCE_FINAL'),
+        )
+        final_subst.update(
+            {
+                'ID_CELL': Foundry.address_TEST_SYMBOLIC(),
+                'ACCOUNTS_CELL': KEVM.accounts(
+                    [
+                        contract,  # test contract address
+                        Foundry.account_CHEATCODE_ADDRESS(KApply('.Map')),
```

Comment:
Is `store_var` supposed to be an empty map in this case? Should we use [`map_empty()`](https://github.com/runtimeverification/pyk/blob/5c625925da8408d691a962f20ef416750b3aebdb/src/pyk/prelude/collections.py#L43), in this case?

Reply:
Yes, I missed that. Thank you!